src/scripts/main.js

```diff
@@ -219,7 +219,17 @@
                     },
                     dataType: "json"
                 }).done(userRqResponse => {
-                    if (debug_logging) {window.console.log({message: "Got user object, invoking huntSend", userRqResponse});}
+                    // The user may have received a King's Reward.
+                    if (userRqResponse.success !== 1 || userRqResponse.puzzle === true) {
```

Comment:
This check of `success` would prevent anyone with an expired session (or for whom we don't correctly forward session auth) from submitting a hunt request (see https://github.com/mh-community-tools/mh-helper-extension/issues/74), so probably we should only use the `puzzle` check.